### ISSUE TITLE
DOC : xfreerdp3 is not available in Debian latest stable release without enabling backports from unstable

### ISSUE THREAD

**Issue by AkechiShiro** [winapps-org/winapps#243]

**Comment by itiligent** [comment]
In case its relevant to mention, the backports repo in debian 12 is enabled by default at install, you'd have to disable it manualy to lose access to freerdp3 packages. Debian 12 also works fine with FreeRDP3 via flatpack. Default sources list in Debian 12 is as follows:

```
cat /etc/apt/sources.list
#deb cdrom:[Debian GNU/Linux 12.1.0 _Bookworm_ - Official amd64 NETINST with firmware 20230722-10:48]/ bookworm main non-free-firmware

deb http://deb.debian.org/debian/ bookworm main contrib non-free non-free-firmware
# deb-src http://deb.debian.org/debian/ bookworm main contrib non-free non-free-firmware

deb http://security.debian.org/debian-security bookworm-security main contrib non-free non-free-firmware
# deb-src http://security.debian.org/debian-security bookworm-security main contrib non-free non-free-firmware

# bookworm-updates, to get updates before a point release is made;
# see https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_updates_and_backports
deb http://deb.debian.org/debian/ bookworm-updates main contrib non-free non-free-firmware
# deb-src http://deb.debian.org/debian/ bookworm-updates main contrib non-free non-free-firmware

# bookworm-backports, previously on backports.debian.org
deb http://deb.debian.org/debian/ bookworm-backports main contrib non-free non-free-firmware
# deb-src http://deb.debian.org/debian/ bookworm-backports main contrib non-free non-free-firmware

# This system was installed using small removable media
# (e.g. netinst, live or single CD). The matching "deb cdrom"
# entries were disabled at the end of the installation process.
# For information about how to configure apt package sources,
# see the sources.list(5) manual.
```

**Comment by AkechiShiro** [comment]
That's not what I have experienced rolling a debian 12 minimal installation inside a VM, are you sure the backports are enabled by default ?

**Comment by itiligent** [comment]
A slight correction as its been a while since I've built with Debian, so I went back and checked. 
If you are looking to make a truly minimal install then you will likely be choosing the advanced install method. Advanced install method includes an option to include backports. Be aware that backports, like many other things in the advanced installer, is not actually selected by default and requires manual selection...
![image](https://github.com/user-attachments/assets/e7fea158-67c1-431b-a0ac-3e70e7689b31)
